RequestManagementSystem/Client/ReqClient.py

```diff
@@ -402,125 +406,138 @@ def resetFailedRequest( self, requestID, allR = False ):
         if op.Status == 'Failed':
           op.Status = 'Waiting'
 
-      return self.putRequest( req )
-    return S_OK( "Not reset" )
+      return self.putRequest(req)
+    return S_OK("Not reset")
 
 #============= Some useful functions to be shared ===========
 
+
 output = ''
-def prettyPrint( mainItem, key = '', offset = 0 ):
+
+
+def prettyPrint(mainItem, key='', offset=0):
   global output
   if key:
     key += ': '
   blanks = offset * ' '
-  if mainItem and type( mainItem ) == type( {} ):
-    output += "%s%s%s\n" % ( blanks, key, '{' ) if blanks or key else ''
-    for key in sorted( mainItem ):
-      prettyPrint( mainItem[key], key = key, offset = offset )
-    output += "%s%s\n" % ( blanks, '}' ) if blanks else ''
-  elif mainItem and type( mainItem ) == type( [] ) or type( mainItem ) == type( tuple() ):
-    output += "%s%s%s\n" % ( blanks, key, '[' if type( mainItem ) == type( [] ) else '(' )
+  if mainItem and isinstance(mainItem, type({})):
```

Comment:
There are multiple replacements for isinstance() which is good but better to make it properly, I mean:

   isinstance(mainItem, type({})) -> isinstance(mainItem, dict)

And similar for other replacements in this file

Reply:
Ah! This is done automatically by autopep8! I though it was doing it properly, probably though only if the original code is `if type(x) == types.dict`.
I will fix them by hand